pkg/networkservice/mechanisms/kernel/kernelvethpair/common.go

```diff
@@ -43,13 +43,13 @@ func appendInterfaceConfig(ctx context.Context, conn *networkservice.Connection,
 		}
 		conf.GetLinuxConfig().Interfaces = append(conf.GetLinuxConfig().Interfaces,
 			&linuxinterfaces.Interface{
-				Name:       name,
+				Name:       name + "-veth",
 				Type:       linuxinterfaces.Interface_VETH,
 				Enabled:    true,
 				HostIfName: linuxName,
 				Link: &linuxinterfaces.Interface_Veth{
 					Veth: &linuxinterfaces.VethLink{
```

Comment:
Also, I would recommend adding these lines from https://github.com/networkservicemesh/networkservicemesh/blob/master/forwarder/vppagent/pkg/converter/kernel_connection_converter.go#L126
```suggestion
				Veth: & linuxinterfaces.VethLink{
					PeerIfName:          name,
					RxChecksumOffloading: linuxinterfaces.VethLink_CHKSM_OFFLOAD_DISABLED,
					TxChecksumOffloading: linuxinterfaces.VethLink_CHKSM_OFFLOAD_DISABLED,
				},
```
this fixes some bugs with packet sending with latest vpp agent

Reply:
Done :)